MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -261,27 +245,26 @@ private async Task AcquireResources(StreamState state, CancellationTokenSource c
             Logger.LogInformation(commandLineLogMessage);
 
             var logFilePrefix = "ffmpeg-transcode";
-            if (state.VideoRequest != null)
+            if (state.VideoRequest != null
+                && string.Equals(state.OutputVideoCodec, "copy", StringComparison.OrdinalIgnoreCase))
             {
-                if (string.Equals(state.OutputVideoCodec, "copy", StringComparison.OrdinalIgnoreCase)
-                    && string.Equals(state.OutputAudioCodec, "copy", StringComparison.OrdinalIgnoreCase))
+                if (string.Equals(state.OutputAudioCodec, "copy", StringComparison.OrdinalIgnoreCase))
                 {
                     logFilePrefix = "ffmpeg-directstream";
                 }
-                else if (string.Equals(state.OutputVideoCodec, "copy", StringComparison.OrdinalIgnoreCase))
+                else
                 {
                     logFilePrefix = "ffmpeg-remux";
                 }
             }
 
             var logFilePath = Path.Combine(ServerConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + "-" + Guid.NewGuid() + ".txt");
-            Directory.CreateDirectory(Path.GetDirectoryName(logFilePath));
```

Comment:
why remove?

Reply:
It's the log dir, there would be a big problem if it didn't exist.